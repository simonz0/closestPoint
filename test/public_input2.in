3
50
90.97 15.28
11.80 17.54
67.72 21.36
53.29 89.14
18.91 95.28
45.37 80.11
48.59 22.27
51.81 49.76
24.80 11.84
45.14 89.81
70.17 81.80
5.99 45.48
43.28 91.11
50.95 77.19
72.25 94.41
40.54 63.21
9.69 52.34
80.75 77.41
73.70 34.04
66.55 92.62
29.32 11.92
72.73 77.91
34.19 24.54
27.66 58.99
36.38 72.80
48.79 6.55
54.60 54.78
52.03 97.88
45.89 2.98
75.07 18.14
97.39 15.61
81.35 7.08
67.95 62.10
84.49 41.65
96.14 51.04
34.27 25.45
62.96 7.00
3.36 97.15
31.54 31.02
56.13 67.92
3.82 4.92
74.47 58.42
59.71 26.51
56.30 5.60
29.49 31.37
23.74 26.88
46.98 5.09
33.96 14.93
67.19 18.44
56.58 63.33
100
69.48 90.85
88.78 32.44
97.85 92.14
29.59 29.39
23.16 85.72
97.31 26.98
90.64 71.78
85.40 50.35
98.29 41.71
55.95 27.78
73.08 79.69
54.66 20.05
84.78 88.62
34.98 51.97
7.06 91.56
15.30 76.55
82.40 4.08
8.99 80.25
96.22 38.57
9.64 19.38
24.29 6.95
46.36 14.94
78.73 31.77
65.29 77.03
73.48 21.24
4.81 46.55
0.93 59.47
66.61 85.71
48.09 1.58
37.68 55.15
93.14 52.98
31.69 75.55
57.06 40.68
55.80 53.28
79.26 65.44
72.66 3.55
72.39 19.02
18.48 51.12
50.79 83.77
28.15 24.27
5.01 32.96
70.82 5.94
92.42 37.43
91.65 40.51
39.01 29.33
95.66 32.15
82.31 27.35
7.70 39.36
68.04 63.50
92.64 47.29
28.94 65.30
50.84 1.33
84.32 69.33
52.45 35.11
53.10 80.60
59.38 58.11
13.55 30.20
64.05 5.98
67.63 55.70
46.49 6.64
85.03 42.15
38.79 67.34
69.50 46.49
6.70 37.54
9.99 99.34
84.83 38.93
64.64 35.67
40.26 48.96
4.99 92.71
84.07 58.09
73.30 43.45
16.20 86.86
73.65 80.26
92.83 41.27
35.96 39.32
47.91 21.00
81.47 86.70
88.34 50.97
33.19 95.04
88.51 43.19
94.39 73.33
82.11 59.03
9.00 22.37
7.99 14.00
15.08 92.06
72.09 88.38
35.51 88.29
75.23 9.15
68.55 68.07
50.43 4.51
7.39 98.34
25.51 88.86
85.04 13.84
39.83 18.23
8.89 28.33
61.42 3.27
1.66 43.53
62.30 10.67
65.90 70.29
24.66 80.98
200
62.35 96.75
69.36 97.86
85.04 44.59
7.01 53.59
12.66 57.44
58.10 20.05
55.78 83.61
8.90 40.82
97.45 48.73
59.05 6.34
77.06 20.47
9.61 78.72
64.01 71.91
89.39 29.91
42.20 14.05
10.89 4.55
10.80 80.25
2.41 95.84
24.84 9.42
49.43 37.50
66.86 7.53
57.55 22.64
91.14 66.45
63.46 88.59
15.18 22.51
94.92 92.24
42.98 4.53
70.96 6.99
76.45 60.35
36.90 18.65
74.40 47.79
23.20 85.20
28.03 25.62
81.04 52.87
35.04 30.47
90.37 1.91
38.00 47.92
24.55 29.14
14.37 88.00
17.73 29.55
10.51 12.65
21.79 53.49
17.19 92.75
60.48 93.63
53.10 97.38
12.28 27.51
45.16 35.49
12.71 73.20
61.10 93.75
26.07 96.15
24.23 16.44
98.05 62.23
64.36 22.60
91.37 78.73
10.60 9.09
8.28 21.12
21.75 30.07
74.61 38.93
22.82 35.09
32.56 75.92
32.47 44.85
3.43 77.64
80.33 16.14
50.84 41.44
9.90 76.91
37.58 34.12
93.35 35.64
96.35 57.71
58.23 87.72
36.44 68.84
96.82 44.72
89.96 18.56
74.79 64.57
57.49 97.60
99.66 90.05
73.53 32.13
34.90 76.96
9.77 15.23
93.10 60.61
56.67 2.99
37.51 94.25
37.12 30.86
29.89 33.47
88.58 88.12
21.19 25.02
56.96 18.01
69.74 46.92
36.57 44.53
11.49 94.06
42.13 11.15
84.12 15.66
43.28 19.02
92.62 53.05
34.25 85.72
13.66 90.92
88.71 51.17
85.18 25.83
82.04 15.06
59.30 70.61
3.19 80.49
95.64 60.15
98.50 65.38
7.07 35.07
9.91 18.56
29.14 52.04
29.70 13.25
67.70 72.99
32.27 60.32
26.04 66.52
46.04 39.70
57.44 34.75
90.87 42.62
60.58 72.90
57.68 19.89
43.52 60.87
0.38 39.15
21.02 98.89
4.53 28.09
33.96 14.44
46.65 63.10
66.48 76.35
76.35 34.18
49.34 8.62
94.51 75.38
75.15 40.55
15.07 32.59
75.30 5.94
75.21 35.89
78.84 32.89
55.77 22.35
93.76 56.15
61.51 14.79
55.04 66.03
42.88 89.00
80.47 89.52
52.09 46.95
65.88 28.45
81.13 15.21
37.07 75.64
90.59 12.22
16.19 5.66
44.81 91.49
11.60 20.02
27.37 90.44
52.91 83.15
12.79 46.67
39.30 74.30
61.46 94.34
40.33 4.34
83.33 20.80
93.86 35.43
67.75 59.74
63.87 48.88
74.95 0.95
24.52 65.54
13.16 40.71
71.20 57.97
32.20 82.79
77.98 59.57
73.23 30.89
42.72 86.02
77.57 82.02
60.32 39.03
76.35 0.65
43.36 59.69
21.45 37.23
95.11 89.20
96.96 58.99
38.08 71.91
59.93 62.60
37.45 73.09
3.31 8.65
31.06 35.51
91.44 9.05
95.08 64.67
39.94 37.79
50.69 17.51
19.81 11.01
56.53 96.16
11.66 99.89
55.85 33.11
37.12 50.96
22.30 34.08
9.95 60.38
6.00 69.88
22.98 43.45
42.98 26.29
52.10 74.04
61.80 43.54
83.09 56.87
8.21 23.03
94.67 58.91
40.53 14.48
69.92 97.07
10.64 81.58
96.96 66.49
14.69 34.08
17.45 36.99
68.16 27.40
97.37 74.16
