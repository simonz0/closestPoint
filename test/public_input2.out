0.9135
22 35
0.8061
10 72
0.4494
5 113
